client/src/helper/utils.ts

```diff
@@ -1,76 +1,81 @@
 import { isString, isInteger } from "lodash-es";
 import { utcToZonedTime } from "date-fns-tz";
 
-export function isIsoDateWithoutTime(string) {
+export function isIsoDateWithoutTime(string: string) {
   const regex = /^\d{4}-(0[0-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0-1])$/g;
   return string?.match(regex) != null;
 }
 
-export function setWindowName(namePostfix) {
+export function setWindowName(namePostfix: string) {
   document.title = `${import.meta.env.VITE_PAGE_TITLE_PREFIX}${namePostfix}`;
 }
-export function retrieveDateOnly(isoDate) {
+export function retrieveDateOnly(isoDate?: string) {
+  if (!isoDate) return "";
   return isoDate.substring(0, 10);
 }
-export function dayAfter(date) {
+export function dayAfter(date?: string) {
+  if (!date) return "";
   const dateObject = new Date(date);
   dateObject.setDate(dateObject.getDate() + 1);
   return retrieveDateOnly(dateObject.toISOString());
 }
 
-export function adjustDateToLocal(originalDate) {
+export function adjustDateToLocal(originalDate: string) {
   const tz = import.meta.env.VITE_BASE_TZ || "Europe/Berlin";
   return utcToZonedTime(new Date(originalDate).getTime(), tz);
 }
 
-export function isEmail(value) {
-  if (!isString(value)) return;
+export function isEmail(value: string) {
+  if (!isString(value)) return; // TODO: Can be removed if all files use TS
```

Comment:
Keine Ahnung, ich habe es selbst lange genug propiert....

Reply:
?

Reply:
Was meinst du?

Reply:
Falscher Kommentar.
Es war "// TODO: How to do this in properly in TS?" gemeint

Reply:
Ich werde es rausfinden;)